assets/charts/components/azure-arc-onboarding/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: azure-arc-onboarding
```

Comment:
hmm, fine choice.

 I went with onboarding because Azure Arc documentation uses `onboarding` to refer to registering a k8s cluster.
 
Azure arc [quickstart](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster) search for onboarding.

Reply:
keeping the name as is and resolving this review comment.